### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by gabrielpoca**
I've broken the build with my last commit.

### CODE REVIEW

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -50,4 +48,4 @@ jobs:
         mix deps.get
 
     - name: Run tests
-      run: mix test
+      run: mix test --timeout 120000
```
Comment (frm) [inline .github/workflows/elixir.yml:51]: What's with this timeout?

Comment (gabrielpoca) [reply .github/workflows/elixir.yml:51]: Since we removed imageflow, transforming images takes much longer.

Comment (naps62) [reply .github/workflows/elixir.yml:51]: can you then open an issue so this doesn't go forgotten then? it feels like somthing that shouldn't be here (even though I know it doesn't really have real impact)

**Review by naps62 — APPROVED** [review]

File: .github/workflows/elixir.yml (hunk 1 above)
Comment (gabrielpoca) [reply .github/workflows/elixir.yml:51]: Done https://github.com/still-ex/still/issues/127

**Pull request merged**